src/utils/parseMetadata.ts

```diff
@@ -412,14 +412,20 @@ function getDatacubeProperties(
               ...(hasLongName(attrs) && { description: attrs.long_name }),
             };
             continue;
-          }
-        }
-        if (hasAxis(attrs)) {
-          const axis = attrs.axis.toLowerCase();
-          if (["x", "y", "z"].includes(axis)) {
+          } else if (hasAxis(attrs)) {
```

Comment:
Argh.. I think I screwed this up. When reading again, I was wondering why there are these `continue` statements. And then I realized, that it's of course fine to have `hasAxis` *and* `hasUnits`, but not `isTimeVariable`. And t would be ok to have and axis without units...

So in short, your previous version likely was more correct in this regard 😬

Reply:
Nice, it's much cleaner now 🙏